### PR TITLE
Cleanaccount

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: htdocs/societe/class/companybankaccount.class.php [hunk 1]
```diff
@@ -193,17 +187,6 @@ class CompanyBankAccount extends Account
 
 	public $country_code;
 
-
-	/**
-	 * @var string owner
-	 */
-	public $proprio;
```
Comment (eldy) [inline htdocs/societe/class/companybankaccount.class.php:200]: To have transition version, we are keeping $this->proprio assignation, for example line 520.
Don't we must keep also the declaration here and add the flag
`@deprecated use $owner`

Comment (frederic34) [reply htdocs/societe/class/companybankaccount.class.php:200]: $this->proprio is in account.class.php and is set to deprecated

Comment (frederic34) [reply htdocs/societe/class/companybankaccount.class.php:200]: https://github.com/Dolibarr/dolibarr/blob/2e938fe9ef32a464aeb0c7f782ec1d261da7f0f5/htdocs/compta/bank/class/account.class.php#L397-L406

Comment (frederic34) [reply htdocs/societe/class/companybankaccount.class.php:200]: if you try to access proprio (private) it will return owner_name

**Pull request merged**